src/tactic/lean_core_docs.lean

```diff
@@ -693,3 +693,79 @@ add_tactic_doc
   category   := doc_category.tactic,
   decl_names := [`tactic.interactive.with_cases],
   tags       := ["core", "combinator"] }
+
+/- conv mode tactics -/
+
+/--
+Navigate to the left-hand-side of a relation.
+A goal of `| a = b` will turn into the the goal `| a`.
+-/
+add_tactic_doc
+{ name       := "to_lhs",
+  category   := doc_category.tactic,
+  decl_names := [`conv.​interactive.​to_lhs],
+  tags       := ["conv"] }
+
+/--
+Navigate to the right-hand-side of a relation.
+A goal of `| a = b` will turn into the the goal `| b`.
+-/
+add_tactic_doc
+{ name       := "to_rhs",
+  category   := doc_category.tactic,
+  decl_names := [`conv.​interactive.​to_rhs],
+  tags       := ["conv"] }
+
+/--
+Navigate into every argument of the current head function.
+A target of `| (a * b) * c` will turn into the two targets `| a * b` and `| c`.
+-/
+add_tactic_doc
+{ name       := "congr",
+  category   := doc_category.tactic,
+  decl_names := [`conv.​interactive.congr],
+  tags       := ["conv"] }
+
+/--
+Navigate into the contents of a `λ` binder.
+A target of `| λ a, a + b` will turn into the targets `| a + b` and introduce `a` into the local
+context.
```

Comment:
If there are multiple binders, all of them will be entered, and if there are none, this tactic is a no-op.